ts/components/messages/MessageList.tsx

```diff
@@ -296,6 +302,27 @@ class MessageList extends React.Component<Props, State> {
     return this.state.itemLayouts[index];
   };
 
+  private onLongPress = (id: string) => {
+    const { messageStates, onLongPressItem } = this.props;
+    onLongPressItem(id);
+    const index = messageStates.findIndex(_ => _.meta.id === id);
+    if (index === messageStates.length - 1) {
```

Comment:
```suggestion
    if (id === messageStates[messageStates.length - 1].meta.id) {
      this.setState({
        longPressedItemIndex: messageStates.length - 1
      });
    }
```
I think we could avoid to use _findIndex_ since we are searching for the last element

Reply:
Done in commit 3cbad4b.